apps/base/utils.py

```diff
@@ -32,3 +33,14 @@ def __call__(self, instance, filename):
         filename = '{}{}'.format(uuid.uuid4(), extension)
         filename = os.path.join(self.path, filename)
         return filename
+
+
+def get_model_object(model_name):
+    def get_model_by_pk(pk):
+        try:
+            model_object = model_name.objects.get(pk=pk)
+            return model_object
+        except model_name.DoesNotExist:
+            raise NotFound('{} {} does not exist'.format(model_name.__name__, pk))
+    get_model_by_pk.__name__ = "get_{}_object".format(model_name.__name__.lower())
```

Comment:
Just a minor glitch over here. Can we please use single quotes instead of double quotes.